frontend/src/reports/OneVariableReport.tsx

```diff
@@ -71,15 +72,17 @@ export function OneVariableReport(props: OneVariableReportProps) {
       : null
   )
 
-  const setVariableConfigWithParam = (v: VariableConfig) => {
+  const isRaceBySex = variableConfig?.variableId.includes(BLACK_WOMEN)
```

Comment:
This is good for now; I think there really be a utility function that includes the logic here: https://github.com/SatcherInstitute/health-equity-tracker/blob/67f955f49977216758e6049632301c18e77f86ca/frontend/src/data/utils/datasetutils.ts#L165

i'd like the functionality to basically look at the metric and determine which special demographic type map to send into the SimpleSelect for demo